corehq/apps/users/bulkupload.py

```diff
@@ -629,7 +648,7 @@ def _load_memoizer(domain):
 
     user_headers, user_rows = parse_users(
         group_memoizer,
-        CommCareUser.by_domain(domain),
+        get_all_commcare_users_by_domain(domain),
```

Comment:
(non-actionable) it could be worth also porting `CommCareUser.by_domain` to call a `dbaccessors` function (I see it's a bit more complicated than what you wrote so would be ok being told this is out of scope)